llvm/test/Feature/Repo/Inputs/remove_GO_ticketnode.py

```diff
@@ -0,0 +1,37 @@
+import argparse
+import re
+
+def filter_definition (line, GOList, filterStr):
+  if (line.startswith("define") and any(f in line for f in GOList)
+        or line.startswith(tuple(GOList))):
```

Comment:
This line is new. Is that on purpose?

Reply:
The line ` line.startswith(tuple(GOList)))` is used to remove the repo_ticketnode from the variable definition. The variable definition starts from its name not from `define`.